oak_functions/examples/mobilenet/client/rust/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
```

Comment:
Was this re-added by mistake?

Reply:
I guess so. I am not sure how it ended up there, but I probably made a mistake when rebasing. Deleted now.